pkg/block/local/adapter.go

```diff
@@ -18,18 +18,21 @@ import (
 	"github.com/google/uuid"
 	"github.com/treeverse/lakefs/pkg/block"
 	"github.com/treeverse/lakefs/pkg/logging"
+	"golang.org/x/exp/slices"
 )
 
 type Adapter struct {
-	path           string
-	removeEmptyDir bool
+	path                    string
+	removeEmptyDir          bool
+	allowedExternalPrefixes []string
```

Comment:
These configurations are missing from the documentation

Reply:
update docs

Reply:
I think removeEmptyDir is not used anywhere, as well as WithRemoveEmptyDir

Reply:
Will cleanup this unused functionality 👍

Reply:
Checked the code - the configuration is not used. But the functionality is 'on' by default.
@N-o-Z do you think we can remove the functionality or just the configuration?